plugin/editorconfig.py

```diff
@@ -8,8 +8,25 @@
         vim.command('let l:ret = 2')
         raise
 
+    # Converts the path to windows when we are in mingw or cygwin
+    def maybe_to_win(filepath):
+        if int(vim.eval("has('win32unix') || has('win64unix')")) != 0:
+            if filepath[0] == '/':
+                # hacky way of making a unix path out of a linux path
+                filepath = filepath[1] + ':' + filepath[2:]
```

Comment:
What do you mean by "making a UNIX path out of a LINUX path"? Are you trying to convert a Cygwin/MinGW path to a Windows path? This might work on MinGW, but on Cygwin, I think a path like `/cygdrive/c/blah` would fail.

Reply:
Bad comment. Should read:

    # hacky way of making a windows path out of a unix path

Yes it will not work with "real" cygwin paths. My aim it to make it work under git bash with vim on windows while keeping it compatible to neovim directly on windows.

Do you have any suggestions?

Reply:
@RedX2501 How about [cygpath](https://cygwin.com/cygwin-ug-net/cygpath.html)?